Dashboard/Views/ConnectionView.axaml

```diff
@@ -64,7 +64,7 @@
         </StackPanel>
     </Grid>
     <!-- Connect Button -->
-    <Button Grid.Row="2" Content="Connect" Margin="0,10,50,50" FontWeight="Bold" HorizontalAlignment="Left"/>
+    <Button Grid.Row="2" Content="Connect" Name="ConnectButton" Margin="0,10,50,50" FontWeight="Bold" Command="{Binding ConnectToSerialPortCommand}" CommandParameter="{Binding SelectedSerialPort}" IsEnabled="True" HorizontalAlignment="Left"/>
         </Grid>
```

Comment:
I can implement a disconnect button.
Also for the above comments
1/2. The File connection and IP connection methods are not implemented, I can disable them and add a disclaimer that they are not implemented right now.
3. We are getting the CarID from the datalogger at the moment. Talking to the AV logger is a stretch goal from my understanding right now.
4. If you launched the app without the sim, you can connect to COM21 since its just a serial port, The connection icon doesn't update since the port is not sending anything back to trigger the RawDataUpdated but I can fix that up so it triggers on initial connection. Currently if a connection actually fails, the app will output the exception to the debug console right now and not change the connect buttons properties.